src/angular/styles/_typography.scss

```diff
@@ -353,45 +305,48 @@ h4,
   @include mqSequence($fontSizes) using ($value) {
     font-size: $value;
   }
-  @include mq($from: desktop, $to: desktop) {
-    line-height: 1.524;
+  @include mq($from: desktop, $to: desktop4k) {
+    line-height: pxToRem(32);
   }
 }
 
 p,
 .sbb-copy {
-  font-size: pxToRem(15);
+  font-size: pxToRem($sizeFontDefault);
   margin: 1em 0 0; // paragraph as default
+  line-height: 1.733;
```

Comment:
I decided to leave the line-heights, when the value was exact and will be used in other sizes as well. Otherwise we would have to add media queries for each size. What do you think?